docs/backend-requests/resources/session-tokens.mdx

```diff
@@ -11,13 +11,13 @@ When a user is authenticated in your application, a short-lived session token is
 
 Below are the default session claims that Clerk generates for you:
 
-* `azp`: authorized party - the `Origin` header that was included in the original Frontend API request made from the user. Most commonly, it will be the URL of the application, e.g. `https://example.com`.
-* `exp`: expiration time - the time at which this token expires.
-* `iat`: issued at - the time at which this token was issued.
-* `iss`: issuer - the Frontend API URL of the instance, e.g. `https://clerk.example.com`.
-* `nbf`: not before - the time before which this token cannot be used.
+* `azp`: authorized party - the `Origin` header that was included in the original Frontend API request made from the user. Most commonly, it will be the URL of the application (e.g. `https://example.com`).
```

Comment:
`azp` could be omitted if for privacy-related reasons we get as `Origin` empty or `null`.